OpenSim/Common/Reporter.h

```diff
@@ -163,10 +163,20 @@ OpenSim_DECLARE_CONCRETE_OBJECT_T(TableReporter_, InputT, Reporter<InputT>);
     TableReporter_() = default;
     virtual ~TableReporter_() = default;
 
-    const TimeSeriesTable_<ValueT>& getReport() const {
+    /** Retrieve the report as a TimeSeriesTable.                             */
+    const TimeSeriesTable_<ValueT>& getTable() const {
```

Comment:
It seems that this could be problematic- giving back a reference to the table that can be wiped out below (`clearTable()`).
I suspect that the crashes that @jimmyDunne is experiencing is because in Java/MATLAB the table reference into the Reporter becomes invalid after `clearTable()` and any use (even *clear* of the MATLAB variable?) would segfault. 

Would it be safer and more inline with expected use cases if `clearTable()` only cleared the data contents (leaving the table and meta data intact) then the reference would remain valid. Are there any downsides to just clearing the matrix data of the `TimeSeriesTable`?  I am curious if that might solve the issue that @jimmyDunne is running into.